From e414d3a9d88c26d0d6cb6f16cb687ee951e8151b Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Mon, 15 Jul 2024 09:45:05 +0200
Subject: [PATCH 1062/1695] dts: stm32: default enable TAMP button on
 stm32mp235f-dk board

Default enable TAMP button on stm32mp235f-dk board. The tamper is in
confirmed mode.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git f2cc965bf39e58015994be1ce6e619a42351aabd dts: stm32: default enable TAMP button on stm32mp235f-dk board]
Change-Id: I07beb2b1184da444c09142b56c661c106ecb8313
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/393232
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/arch/arm/dts/stm32mp235f-dk.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/dts/stm32mp235f-dk.dts b/core/arch/arm/dts/stm32mp235f-dk.dts
index 5141d0eb7..85d47f1ba 100644
--- a/core/arch/arm/dts/stm32mp235f-dk.dts
+++ b/core/arch/arm/dts/stm32mp235f-dk.dts
@@ -458,7 +458,7 @@
 
 	/* Tamper button */
 	tamp-button {
-		status = "disabled";
+		status = "okay";
 		tamper-gpios = <&gpioz 2 0>;
 		st,tamp-mode = <TAMPER_CONFIRMED_MODE>;
 		st,tamp-id = <3>;
-- 
2.47.2

